docs/source/concepts/glossary.rst

```diff
@@ -393,24 +456,92 @@ As a user, you don't need to interact with Daml-LF directly. But internally, it'
 - sending and receiving values via the Ledger API (using a protocol such as gRPC)
 - generating code in other languages for interacting with Daml models (often called “codegen”)
 
-General concepts
-****************
+Composability
+=============
+
+Composability is the ability of a participant to extend an existing system with new Daml applications or new topologies unilaterally without requiring cooperation from anyone except the 
+directly involved participants who wish to be part of the new application functionality.
+
+
 
-Ledger, Daml ledger
-===================
 
-**Ledger** can refer to a lot of things, but a ledger is essentially the underlying storage mechanism for a running Daml applications: it's where the contracts live. A **Daml ledger** is a ledger that you can store Daml contracts on, because it implements the `ledger API <#ledger-api>`__.
 
-Daml ledgers provide various guarantees about what you can expect from it, all laid out in the :doc:`/concepts/ledger-model/index` page.
+Canton Concepts
+***************
+
+.. Language for the participant node was taken, and edited, from the Canton key terms page
```

Comment:
drop these comments please

Reply:
I will, but why so offended?  You've seen them so you know the context to the sentences, but no one else will.

Reply:
Sorry this came across the wrong way. I’m not all offended. The reason why I want to drop them is that I generally want our repo to document the current state and not so much the history of how we got there because that quickly gets out of date and shouldn’t really be relevant.